### PR TITLE
Patch spawn positions

### PR DESCRIPTION
**Pull request by simonkellly**
Patched it so that you spawn in the normal spawn locations as if you have playerId mod 10

### CODE REVIEW

**Comment by DorCoMaNdO** [comment]
No need to have Prefix with a bool return value if you're always going to return true, you can change it to void and remove the return.

File: CrowdedMod/Patches/GenericPatches.cs [hunk 1]
```diff
@@ -46,6 +46,17 @@ static class PlayerTabUpdateAvailableColorsPatch {
 				return false;
 			}
 		}
+		
+		[HarmonyPatch(typeof(ShipStatus), nameof(ShipStatus.GetSpawnLocation))]
+        	public static class ShipStatusGetSpawnLocationPatch
+		{
+		    public static bool Prefix(ShipStatus __instance, [HarmonyArgument(0)] ref int playerId, [HarmonyArgument(1)] ref int numPlayer, [HarmonyArgument(2)] bool initialSpawn)
```
Comment (Galster-dev) [inline CrowdedMod/Patches/GenericPatches.cs]: you can leave only required arguments here, harmony will patch it anyway (unless it's overrided)

File: CrowdedMod/Patches/GenericPatches.cs [hunk 2]
```diff
@@ -46,6 +46,17 @@ static class PlayerTabUpdateAvailableColorsPatch {
...
+		    {
+			playerId %= 10;
+			if (numPlayer >= 10) numPlayer = 10;
+			return true;
```
Comment (Galster-dev) [inline CrowdedMod/Patches/GenericPatches.cs]: you don't need to use bool-prefix if it always returns true. void-prefix will do the same by [comment](https://github.com/CrowdedMods/CrowdedMod/pull/19#issuecomment-770851779)

File: CrowdedMod/Patches/GenericPatches.cs [hunk 3]
```diff
@@ -46,6 +46,17 @@ static class PlayerTabUpdateAvailableColorsPatch {
...
 			}
 		}
+		
+		[HarmonyPatch(typeof(ShipStatus), nameof(ShipStatus.GetSpawnLocation))]
+        	public static class ShipStatusGetSpawnLocationPatch
+		{
+		    public static bool Prefix(ShipStatus __instance, [HarmonyArgument(0)] ref int playerId, [HarmonyArgument(1)] ref int numPlayer, [HarmonyArgument(2)] bool initialSpawn)
+		    {
+			playerId %= 10;
+			if (numPlayer >= 10) numPlayer = 10;
```
Comment (Galster-dev) [inline CrowdedMod/Patches/GenericPatches.cs]: why `>=`, not just `>`?

**Review by Galster-dev — CHANGES_REQUESTED** [review]
very hard review !

**Review by Galster-dev — APPROVED** [review]

**Pull request merged**